packages/rest/package.json

```diff
@@ -26,7 +26,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "class-transformer": "^0.4.0",
+    "class-transformer": "0.5.1",
     "class-validator": "^0.13.1",
```

Comment:
We need to lock it to `0.13.1` (without the caret `^`). Otherwise it will automatically download `0.13.2` still

```suggestion
    "class-validator": "0.13.1",
```